### ISSUE TITLE
Error: unable to fetch the response from the backend: unexpected EOF

### ISSUE THREAD

**Issue by COil** [symfony-cli/symfony-cli#535]
Sometimes when I switch from a project to another, I have the following error : 

<img width="676" alt="Capture d’écran 2024-11-04 à 06 50 30" src="https://github.com/user-attachments/assets/dbacd22c-892f-47f3-8254-0e96dce75a23">

I must reboot to make things works again, restarting the Symfony CLI does not work.

**Comment by tucksaun** [comment]
hi @COil 

could you elaborate when you say "switch from a project to another"?

**Comment by COil** [comment]
```
cd ~/project1
symfony serve --daemon --port=8000
symfony server:stop
```

```
cd ~/project2
symfony serve --daemon --port=8000
```

Is there a way to have a more detailed log the next I see this error?

**Comment by tucksaun** [comment]
You can run `symfony serve --daemon=false -vvv --port=8000` instead. it should give us plenty of logs to work with

**Comment by COil** [comment]
OK, thanks, I will try that. Let's close this issue for now. I'll create a new ticket with all the info.

**Comment by COil** [comment]
Hello I have this error 

```
cd ~/project1
symfony serve --daemon --port=8000
symfony server:stop
```

```
cd ~/project2
symfony serve --port=8000
open https://127.0.0.1:8000/

> [Application] Nov 10 06:49:42 |CRITICA| REQUES Uncaught PHP Exception ErrorException: "Warning: session_start(): Class 'App\Enum\EasterEggDifficulty' not found" at NativeSessionStorage.php line 172

But `App\Enum\EasterEggDifficulty'` is a class of project 1, I don't understand why it tries to use it.

**Comment by tucksaun** [comment]
It seems like you have session clashes.

Project 1 would store some `EasterEggDifficulty` serialized in the session and then when the session is reloaded in project2 (because you have the same IP, port and cookie name) it can not unserialized the data because the enum is not known.

Using the proxy and the domain names or different cookie names should fix this.

**Comment by COil** [comment]
I agree with the root cause, but we should have a standart 500 error, not this error "unable to fetch data from the backend".

**Comment by tucksaun** [comment]
what do you mean by an "standard 500 error"?

**Comment by COil** [comment]
The error displayed by the application, it is the screenshot of the first message.

**Comment by tucksaun** [comment]
But it looks like the application does not send any error and PHP crashes instead that's why we show this error.
